app/controllers/archived_vouchers_controller.rb

```diff
@@ -0,0 +1,20 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+class ArchivedVouchersController < VouchersController
+  def index
+    @show_search = true
+    @vouchers = @subprogram.vouchers.order(:id)
+    @vouchers_for_page = @vouchers.archived
+    @voucher_state = 'archived'
+  end
+
+  def restore
+    @voucher.update(archived_at: nil)
```

Comment:
I think we might want to set `available` to false here and `date_available` to nil.  That way it won't auto match if it somehow ended up as archived but "available"

Reply:
This has been added to the PR 
https://github.com/greenriver/boston-cas/pull/867/commits/da6e66bb3238941ad95a54c5e1be9a2cfe6aa485